### PR TITLE
Added play/pauseAud functions, new bounceSound example.

### PR DESCRIPTION
**Pull request by nshaheed**
I've added the functions play and pauseAud to handle the basic playback elements for audio (addressing part 1 of issue #61).  This involved adding an `AudMethod` parameter to the `Canvas` data type to facilitate the `<audio>.<function>` format.

Additionally, I've adapted the `bounce` example to `bounceSound` to include a short sound effect whenever a ball hits the bottom of the canvas

### CODE REVIEW

File: Graphics/Blank.hs [hunk 1]
```diff
@@ -99,8 +99,11 @@ module Graphics.Blank
          -- ** 'CanvasContext', and off-screen Canvas.
         , CanvasContext
         , newCanvas
-        , newAudio --NickS addition
-        , AudioInfo -- NickS addition
+          -- ** Audio functionality
+        , newAudio 
+        , AudioInfo
+        , play
+        , pauseAud
```
Comment (RyanGlScott) [inline Graphics/Blank.hs]: There some inconsistency in the naming conventions of audio-related functions right now. For consistency with the image-related functions, let's use the word `Audio`, fully spelled out, with these names (i.e., name them `playAudio` and `pauseAudio`).

File: Graphics/Blank/Canvas.hs [hunk 2]
```diff
@@ -103,6 +104,10 @@ data Method
         | Transform (Double, Double, Double, Double, Double, Double)
         | Translate (Double, Double)
 
+data AudMethod
+        = forall audio . Audio audio => Play audio
+        | forall audio . Audio audio => Pause audio
+
```
Comment (RyanGlScott) [inline Graphics/Blank/Canvas.hs]: Similarly, let's name this `AudioMethod`.

**Comment by RyanGlScott** [comment]
This looks good, save for some semantic quibbles that I noted above. The only other suggestion I could make is that at the moment, the `audio` example isn't very interesting to a casual observer. Since we now can play/pause audio, why not implement a playback bar with basic functionality? Later, we can add things like volume as well.

**Comment by nshaheed** [comment]
I've updated everything to follow the processAudio convention (including audio examples, etc.). Next, time to work on the playbar.

**Comment by nshaheed** [comment]
I've updated this branch to be up to date, and I've added a few other functions (setLoopAudio, setMutedAudio, and setPlaybackRateAudio.

**Pull request merged**